mission_control/src/mission_control.cpp

```diff
@@ -278,7 +280,14 @@ void MissionControlNode::queryMissionsCallback(const mission_control::QueryMissi
 
   report_missions_pub_.publish(msg);
 }
-
+void MissionControlNode::stopMissionCallback(const std_msgs::Bool& msg)
+{
+  if (current_mission_ && msg.data)
+  {
+    reportOn(current_mission_->preempt());
+    ROS_DEBUG_STREAM("stopMissionsCallback - Stopping mission");
+  }
+}
```

Comment:
@patapufate meta: I'm not thrilled with having some interfaces failing and some interfaces being a no-op if no mission is active. We should homogenize this later on.